### DISCUSSION TITLE
Support for rendering github emoji markup as an equivilent emoji character

### DISCUSSION THREAD

**Discussion by johnhamelink** [magit/forge#691] [Ideas]
Github markdown contains the notion of emoji defined with a colon prefix & suffix (such as `:+1:` which is rendered as the thumbs up emoji 👍🏼). It would be nice to render the emoji syntax when viewing topics.

One example where this would be particularly helpful is where emoji is assumed to be one char in size, such as in this CI output comment which produces test result information:

```
## Test Results
    4 files      4 suites   40m 41s :stopwatch:
9 096 tests 9 087 :white_check_mark: 9 :zzz: 0 :x:
9 235 runs  9 226 :white_check_mark: 9 :zzz: 0 :x:
```

Which when rendered resolves to:

```
Test Results

    4 files      4 suites   40m 41s ⏱️
9 096 tests 9 087 ✅ 9 💤 0 ❌
9 235 runs  9 226 ✅ 9 💤 0 ❌
```

And on forge 0.4.0 the topic renders like so in Emacs 31.0.50:

![Emacs](https://github.com/user-attachments/assets/e2d0aefc-2ad7-43dc-bb07-225d84659f2a)

---

**EDIT**: Turns out you can enable it as follows:

```emacs-lisp
(use-package emojify :ensure t)
(use-package forge
  :ensure t
  :after (magit emojify)
  :hook (forge-topic-mode . emojify-mode))
```

**Comment by tarsius** [comment]
Please use the [`emojify`](https://github.com/iqbalansari/emacs-emojify) package for that.
